tests/benchmarks/text-0.11.2.3/Data/Text/Encoding.hs

```diff
@@ -264,8 +264,8 @@ encodeUtf8 (Text arr off len) = unsafePerformIO $ do
                         else do
                             let v = A.unsafeIndex arr t
                             if v >= 0x80 then go d' t (u `minusPtr` ptr) else do
-                            poke u (fromIntegral v :: Word8)
-                            ascii (d'-1) (t+1) (u `plusPtr` 1)
+                              poke u (fromIntegral v :: Word8)
```

Comment:
Don't know how this ever worked: it's interpreted as an empty do block by GHC without the indent...